packages/app/package.json

```diff
@@ -9,21 +9,22 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@affine/component": "workspace:*",
     "@affine/datacenter": "workspace:*",
     "@affine/i18n": "workspace:*",
     "@blocksuite/blocks": "0.4.0-alpha.2",
     "@blocksuite/editor": "0.4.0-alpha.2",
     "@blocksuite/icons": "^2.0.2",
     "@blocksuite/store": "0.4.0-alpha.2",
-    "@emotion/css": "^11.10.0",
-    "@emotion/react": "^11.10.4",
+    "@emotion/css": "^11.10.5",
+    "@emotion/react": "^11.10.5",
     "@emotion/server": "^11.10.0",
-    "@emotion/styled": "^11.10.4",
+    "@emotion/styled": "^11.10.5",
     "@fontsource/poppins": "^4.5.10",
     "@fontsource/space-mono": "^4.5.10",
-    "@mui/base": "^5.0.0-alpha.87",
-    "@mui/icons-material": "^5.10.9",
-    "@mui/material": "^5.8.6",
+    "@mui/base": "=5.0.0-alpha.101",
+    "@mui/icons-material": "=5.10.9",
+    "@mui/material": "=5.8.6",
```

Comment:
there is a breaking change in 5.11.x

Reply:
its recommend to submit it as an issue and fix it as soon as possible to avoid version locking